### PR TITLE
Edit and listed yaml files, Add task in Makefile

### PR DESCRIPTION
**Pull request by anniepamela2020**
The yaml files in the project were not respecting the correct syntax of yaml files. Used yamllint to lint and format the yaml files throughout the project.

Fixes Yaml files syntax

### CODE REVIEW

File: examples/ibm-tutorial/build-and-deploy-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    #       workspace: git-source
+    #   params:
+    #     - name: pathToYamlFile
+    #       value: $(params.pathToYamlFile)
+    #     - name: imageUrl
+    #       value: $(params.imageUrl)
+    #     - name: imageTag
+    #       value: $(params.imageTag)
+    #     - name: imageDigest
+    #       value: $(tasks.source-to-image.results.IMAGE-DIGEST)
```
Comment (bradbeck) [inline examples/ibm-tutorial/build-and-deploy-pipeline.yaml:95]: I believe this has been converted to CUE and no longer exists as YAML

File: examples/ibm-tutorial/pipeline-account.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
```
Comment (bradbeck) [inline examples/ibm-tutorial/pipeline-account.yaml:1]: I believe this has been converted to CUE and no longer exists as YAML

File: examples/ibm-tutorial/pipeline-pvc.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (bradbeck) [inline examples/ibm-tutorial/pipeline-pvc.yaml:1]: I believe this has been converted to CUE and no longer exists as YAML

File: examples/ibm-tutorial/pipeline-run.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (bradbeck) [inline examples/ibm-tutorial/pipeline-run.yaml:1]: I believe this has been converted to CUE and no longer exists as YAML

File: examples/sample-pipeline/pipeline-account.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (bradbeck) [inline examples/sample-pipeline/pipeline-account.yaml:1]: I believe this has been converted to CUE and no longer exists as YAML

File: examples/sample-pipeline/registry-secret.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (bradbeck) [inline examples/sample-pipeline/registry-secret.yaml:1]: I believe this has been converted to CUE and no longer exists as YAML

File: examples/sample-pipeline/ssf-pipeline.yaml [hunk 7]
```diff
@@ -0,0 +1,92 @@
+---
```
Comment (bradbeck) [inline examples/sample-pipeline/ssf-pipeline.yaml:1]: I believe this has been converted to CUE and no longer exists as YAML

File: examples/sample-pipeline/ssf-run.yaml [hunk 8]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (bradbeck) [inline examples/sample-pipeline/ssf-run.yaml:1]: I believe this has been converted to CUE and no longer exists as YAML

File: examples/sample-pipeline/task-build-push-image.yaml [hunk 9]
```diff
@@ -0,0 +1,72 @@
+---
```
Comment (bradbeck) [inline examples/sample-pipeline/task-build-push-image.yaml:1]: I believe this has been converted to CUE and no longer exists as YAML

File: examples/sample-pipeline/task-deploy.yaml [hunk 10]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (bradbeck) [inline examples/sample-pipeline/task-deploy.yaml:1]: I believe this has been converted to CUE and no longer exists as YAML

File: examples/sample-pipeline/task-git-clone.yaml [hunk 11]
```diff
@@ -0,0 +1,233 @@
+---
```
Comment (bradbeck) [inline examples/sample-pipeline/task-git-clone.yaml:1]: I believe this has been converted to CUE and no longer exists as YAML

File: examples/sample-pipeline/task-gyrpe-scan.yaml [hunk 12]
```diff
@@ -0,0 +1,51 @@
+---
```
Comment (bradbeck) [inline examples/sample-pipeline/task-gyrpe-scan.yaml:1]: I believe this has been converted to CUE and no longer exists as YAML

File: examples/sample-pipeline/task-syft-bom-generation.yaml [hunk 13]
```diff
@@ -0,0 +1,64 @@
+---
```
Comment (bradbeck) [inline examples/sample-pipeline/task-syft-bom-generation.yaml:1]: I believe this has been converted to CUE and no longer exists as YAML

File: platform/components/tekton/chains/patch_config_intoto.yaml [hunk 14]
```diff
@@ -0,0 +1,3 @@
+---
```
Comment (bradbeck) [inline platform/components/tekton/chains/patch_config_intoto.yaml:1]: I believe this was removed in a recent PR

**Comment by bradbeck** [comment]
There are several `.DS_Store` files that I don't believe we want included.

File: .github/workflows/install-ssf.yaml [hunk 15]
```diff
@@ -14,28 +17,52 @@ jobs:
...
+- name: Run test pipeline
+  run: |
+    ./examples/buildpacks/buildpacks.sh
+    export DOCKER_IMG=$(tkn pr describe --last -o jsonpath='{.spec.params[?(@.name=="APP_IMAGE")].value}')
+    tkn pr logs --last -f
+    sleep 60
+    docker run --rm gcr.io/go-containerregistry/crane ls $DOCKER_IMG
+    tkn tr describe --last -o json | jq -r '.metadata.annotations["chains.tekton.dev/signed"]'
+    cosign verify --key k8s://tekton-chains/signing-secrets ${DOCKER_IMG}
+    cosign verify-attestation --key k8s://tekton-chains/signing-secrets ${DOCKER_IMG}
```
Comment (bradbeck) [inline .github/workflows/install-ssf.yaml:68]: This seems to be from an older revision of this file before `make setup-kyverno` was split out into its own step.

**Review by bradbeck — DISMISSED** [review]

File: Makefile [hunk 16]
```diff
@@ -80,3 +80,9 @@ docs-build: ## Build the documentation site
 .PHONY: linter-markdown
 linter-markdown: ## Lint markdown files
 	npx markdownlint-cli2  "**/*.md" "#docs"
+
+.PHONY: lint-yaml
+lint-yaml: ## Lint yaml files
+	cd resources && yamllint .
+	cd examples && yamllint .
+	cd platform && yamllint .
```
Comment (rgreinho) [inline Makefile:88]: Quick question: why not simply running `yamllint .` here, and ignoring the paths you don't want to lint in the configuration file?

**Review by pxp928 — COMMENTED** [review]
Should we squash the commits into 1 to clean up this PR?

**Review by bradbeck — APPROVED** [review]

**Review by trmiller — APPROVED** [review]
LGTM

**Review by pxp928 — APPROVED** [review]

**Pull request merged**